plugins/DNS/plugin.go

```diff
@@ -0,0 +1,53 @@
+package dns
+
+import (
+	"errors"
+	"net"
+
+	"github.com/coredhcp/coredhcp/handler"
+	"github.com/coredhcp/coredhcp/logger"
+	"github.com/coredhcp/coredhcp/plugins"
+	"github.com/insomniacslk/dhcp/dhcpv4"
+	"github.com/insomniacslk/dhcp/dhcpv6"
+)
+
+var (
+	dnsServers []net.IP
```

Comment:
Can this variable be moved below the init function to make it consistent with other plugins?